bundle/manifests/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -50,13 +50,28 @@ spec:
         kind: CommonService
         name: commonservices.operator.ibm.com
         specDescriptors:
-          - description: 'Accepting the license - URL: https://ibm.biz/integration-licenses'
+          - displayName: License
+            path: license
+            description: License information for this instance. You must accept the license.
+          - description: Read and accept the license that is applicable to your installation. For more information, see https://ibm.biz/integration-licenses'
```

Comment:
```suggestion
          - description: Read and accept the license that is applicable to your installation. For more information, see https://ibm.biz/TBD'
```
Needs the new URL

Reply:
This is covered by https://github.com/IBM/ibm-common-service-operator/pull/1621 - pls remove the license link from this PR and focus only on the other proposed changes. Thanks!